packages/web-app-files/src/components/SideBar/Shares/SpaceMembers.vue

```diff
@@ -141,6 +142,7 @@ export default defineComponent({
         })
         .catch((error) => {
           console.error(error)
+          this.hideModal()
```

Comment:
Can we please take a finally and do the hideModal call there ?

Reply:
done 👍